src/Compilers/Core/Portable/PublicAPI.Unshipped.txt

```diff
@@ -55,10 +55,9 @@ Microsoft.CodeAnalysis.NullabilityInfo.Annotation.get -> Microsoft.CodeAnalysis.
 Microsoft.CodeAnalysis.NullabilityInfo.Equals(Microsoft.CodeAnalysis.NullabilityInfo other) -> bool
 Microsoft.CodeAnalysis.NullabilityInfo.FlowState.get -> Microsoft.CodeAnalysis.NullableFlowState
 Microsoft.CodeAnalysis.NullableAnnotation
-Microsoft.CodeAnalysis.NullableAnnotation.Annotated = 3 -> Microsoft.CodeAnalysis.NullableAnnotation
-Microsoft.CodeAnalysis.NullableAnnotation.Disabled = 1 -> Microsoft.CodeAnalysis.NullableAnnotation
```

Comment:
>Disabled [](start = 42, length = 8)

Isn't this API already released publicly?

Reply:
It's released as a preview API, which we have previously said was subject to change. This is the last chance we have to make breaking changes here.

---
In reply to: [306591815](https://github.com/dotnet/roslyn/pull/37442#discussion_r306591815) [](ancestors = 306591815)